packages/instrumented-component-lifecycle/rollup.config.mjs

```diff
@@ -0,0 +1,19 @@
+import createRollupConfig from '../../.rollup/index.mjs'
+import noTreeShakingPlugin from '../../.rollup/no-tree-shaking.plugin.mjs'
+import jsx from 'acorn-jsx'
+
+const config = createRollupConfig({
+  external: [
+    '@bugsnag/browser-performance',
+    '@bugsnag/core-performance',
```

Comment:
I think `@bugsnag/core-performance` is already included as part of `createRollupConfig`